shell/components/Tabbed/index.vue

```diff
@@ -557,6 +557,7 @@ export default {
       list-style: none;
       padding: 0;
       margin-top: auto;
+      z-index: 1;
```

Comment:
Can we make use of the pre-defined z-indexes specified in shell/assets/styles/base/_variables.scss?